### PR TITLE
utils/openocd_cmd: add support for all available open nodes

### PR DESCRIPTION
**Pull request by aabadie**
This PR allows to easily flash any kind of open nodes that support openocd from a gateway.

### CODE REVIEW

**Comment by rienafairefr** [comment]
You could use the node plugin registry instead of hardcoding the nodes? Then you can check if the class is subtype of `NodeOpenOcd`.

**Comment by aabadie** [comment]
> You could use the node plugin registry instead of hardcoding the nodes? Then you can check if the class is subtype of NodeOpenOcd.

Thinking of it and I like the idea. I'll give it a try.

**Comment by aabadie** [comment]
@rienafairefr, I updated the PR like you suggested also applied the same logic for avrdude based nodes.
I tested with an M3 and ST Lrwan1 board and it works fine. Even without the try/except in the automatic nodes module importer logic, so I also removed it!

File: gateway_code/nodes.py [hunk 1]
```diff
@@ -116,10 +116,7 @@ def import_all_nodes(pkg_dir):
     for (module_loader, name, _) in pkgutil.iter_modules([pkg_dir]):
         if name in ['tests', 'common']:
             continue
-        try:
-            module_loader.find_module(name).load_module(name)
-        except ImportError:
-            pass
+        module_loader.find_module(name).load_module(name)
```
Comment (rienafairefr) [inline gateway_code/nodes.py:119]: weird, but good, that it now works :-)

**Comment by rienafairefr** [comment]
Did you test on an avr based board as well ?

**Comment by aabadie** [comment]
> Did you test on an avr based board as well ?

No. It's possible on one of the CI node (there's a zigduino on Strasbourg and a Leonardo in Grenoble).

**Comment by aabadie** [comment]
@rienafairefr, I tested on leonardo and zigduino and it also works. I think we can merge :)

**Pull request merged**